tester/src/main.rs

```diff
@@ -95,7 +95,7 @@ async fn main() -> Result<(), Error> {
     logger_init()?;
 
     let mut entries = Vec::new();
-    let mut dir = fs::read_dir("json").await?;
+    let mut dir = fs::read_dir("tester/json").await?;
```

Comment:
I think I'd rather do a `cd` in the CI than change that here tbh 🤔

Reply:
Where in the CI do the `cd`?

Reply:
Before you clone hornet I think

Reply:
What's the status with that ?

Reply:
Didn't work

Reply:
:carlos: